kpatch-build/create-diff-object.c

```diff
@@ -69,6 +69,8 @@
 		printf(format, ##__VA_ARGS__); \
 })
 
+extern char *find_exported_symbol_objname(const char *fname, char *symname);
```

Comment:
This function is kind of in the same realm as the functions in lookup.c, though the lookup functions look at the vmlinux or original module file rather than Module.symvers.  I wonder if it would be better to abstract this functionality in lookup.c?  lookup_open() could take an additional argument (the path to Module.symvers) and then there could be a lookup_exported_symbol() or something.

Or alternatively,